homeassistant/components/rako/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""The Rako integration."""
+from __future__ import annotations
+
+from asyncio import Task
+import logging
+from typing import TypedDict
+
+from homeassistant.components.light import DOMAIN as LIGHT_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+
+from .bridge import RakoBridge
+from .const import DOMAIN
+from .light import RakoLight
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RakoDomainEntryData(TypedDict):
+    """A single Rako config entry's data."""
+
+    rako_bridge_client: RakoBridge
+    rako_light_map: dict[str, RakoLight]
+    rako_listener_task: Task | None
```

Comment:
Move it to `model.py`. With that you won't have circular dependencies for this one.